tests/unit/models/360-core.js

```diff
@@ -149,6 +149,14 @@ describe("SerSol 360 Core Model >", function() {
       expect(data).toBeDefined();
       expect(serSol360Core.getBrowZineWebLink(data)).toEqual("https://browzine.com/libraries/XXX/journals/10292");
     });
+
+    it("should return null when the BrowZine API response for a journal is empty", function() {
+      var data = serSol360Core.getData({
+        "data": []
```

Comment:
These tests make my heart warm with confidence that we won't regress on handling these cases 😊 ❤️

Reply:
It's a beautiful thing, indeed.